### ISSUE TITLE
Nested heredoc trouble

### ISSUE THREAD

**Issue by garthk** [rrrene/credo#781]
### Environment

* Credo version (`mix credo -v`): 1.4.0
* Erlang/Elixir version (`elixir -v`): Elixir 1.10.3 on Erlang 22.3.4.1
* Operating system: macOS

### What were you trying to do?

`lib/heredoc-demo.ex`:
```elixir
defmodule HereDocDemo do
  @moduledoc """
  Demonstrates a bug.

      $ mix credo lib/heredoc-demo.ex

      warning: outdented heredoc line. The contents inside the heredoc should be indented at the
      same level as the closing '''. The following is forbidden:

          ...

      Instead make sure the contents are indented as much as the heredoc closing:

          ...

      The current heredoc line is indented too little
        lib/heredoc-demo.ex:25
  """

  @doc ~S'''
  José suggested using an outer sigil so the inner sigil was more normal in the documentation

    ~E"""
    but Credo didn't like it at all
    """
  '''
  def demo, do: :ok
end
```

### Expected outcome

No error, because the `~E"""` inside the doc-string is appropriately indented.

### Actual outcome

      The current heredoc line is indented too little
        lib/heredoc-demo.ex:25

**Comment by rrrene** [comment]
I am looking into this (I misread this issue at first and went down an unnecessary rabbit hole, but am on track again).

**Comment by rrrene** [comment]
This is now fixed in `v1.6.3`. Thx again and sorry for the delay!
